src/popup/router/components/BoxButton.vue

```diff
@@ -68,6 +68,21 @@ export default {
     }
   }
 
+  &.light {
+    &:hover {
+      background: rgba(variables.$color-white, 0.1);
+
+      &,
+      ::v-deep svg {
```

Comment:
Please avoid styling child component's inner code (v-deep).

Reply:
If something is placed inside of the slot you can just use it's class. In this case `.icon` as all the icons has it.